pkg/protobuf/app/app.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package app;
+
+service App {
+    rpc Create(CreateRequest) returns (Empty);
+}
+
+message CreateRequest {
+    string name = 1;
+    string team = 2;
+    string process_type = 3;
+
+    message Limits {
+        message LimitRangeQuantity {
+            string quantity = 1;
+            string resource = 2;
+        }
+  
+        repeated LimitRangeQuantity default = 1;
+        repeated LimitRangeQuantity default_request = 2;
+        repeated LimitRangeQuantity limit_request_ratio= 3;
+        repeated LimitRangeQuantity max = 4;
+        repeated LimitRangeQuantity min = 5;
```

Comment:
We talked about this fields and we decided to remove and simplify limits to `default` and `default_request`.